packages/libs/react-ui/src/styles/sprinkles.css.ts

```diff
@@ -13,6 +13,7 @@ export const breakpoints: Record<string, string> = {
 const systemProperties = defineProperties({
   properties: {
     fontFamily: vars.fonts,
+    borderSpacing: vars.sizes,
```

Comment:
Looks like border-spacing is a table specific css property so it likely won't be used much. I would remove this from sprinkles